packages/messages/src/browser/notifications-message-client.ts

```diff
@@ -0,0 +1,55 @@
+/*
+* Copyright (C) 2017 TypeFox and others.
```

Comment:
missing spaces in copyright, wish to have tslint rule for all missing and misformatted copyrights :(